packages/devtools_app/lib/src/screens/vm_developer/vm_service_private_extensions.dart

```diff
@@ -9,6 +9,9 @@ import 'package:vm_service/vm_service.dart';
 /// fashion. Objects and extensions in this class should not be used outside of
 /// the `vm_developer` directory.
 
+const referencesKey = 'references';
```

Comment:
These can be included directly in the `extension` if you make them `static`.

Reply:
Thanks!